src/backend/cdb/motion/ic_tcp.c

```diff
@@ -1775,7 +1775,7 @@ SetupTCPInterconnect(EState *estate)
 		i = 0;
 		while (n > 0 &&
 			   outgoing_count < expectedTotalOutgoing &&
-			   i < sendingChunkTransportState->numConns)
+			   i < sendingChunkTransportState ? sendingChunkTransportState->numConns : 0)
```

Comment:
Pointer sendingChunkTransportState may be dereferenced [below](https://github.com/arenadata/gpdb/pull/768/files#diff-8de7e292e2ab3f0d6b1229dd48de95c2589b047830aab7f3777cf8de0ef24bf1R1780)